### ISSUE TITLE
BUG: Standalone mode template columns not working properly 

### ISSUE THREAD

**Issue by MaxStubbeGasunie** [marianfoo/ui5-cc-spreadsheetimporter#364]
Im using the standalone version of the spreatsheetbutton 

when i  give the property columns value "Id,Time,Value"
The template gives back an excel that only contains the last column. 
So the template only contains one column "Value"

**Comment by marianfoo** [comment]
So the problem is only the template?
Please share the whole config

**Comment by daniellelyle** [comment]
@marianfoo I also came across this issue. Here is my `componentData`:

```
componentData: {
                    columns: ["field", "field2"],
                    standalone: true,
                    hidePreview: true
                }
```

No matter how many columns are added to the array, the template only provides the last one.

**Comment by marianfoo** [comment]
fixed in #369 
ui5-cc-spreadsheetimporter: 0.25.4
ui5-cc-spreadsheetimporter-button: 0.11.4
